docs/deprecated.md

```diff
@@ -0,0 +1,37 @@
+# Deprecated functionality
```

Comment:
Is this document linked from the main README.md? If not, it should

Reply:
Fixed with 1bbaed44caf152c9cd022154525162218accdfab and 6f89145c46d949c6ae37b4ac499313c28bf9449e